doc/guacamole-playback-example/src/main/webapp/index.html

```diff
@@ -28,10 +28,19 @@
     <body>
 
         <!-- Guacamole recording player -->
-        <div class="player">
+        <div id="player">
```

Comment:
Not that I have a problem with making this an ID, but you could have used getElementsByClassName instead of changing this, right?